include/unifex/transform.hpp

```diff
@@ -170,6 +171,7 @@ struct _sender<Predecessor, Func>::type {
 
   template <typename Receiver>
   auto connect(Receiver&& r) &
+      noexcept(is_nothrow_tag_invocable_v<tag_t<unifex::connect>, Predecessor&, receiver<std::remove_cvref_t<Receiver>>>)
```

Comment:
Also needs to check that both `Receiver` and `Func` is nothrow-decay-copyable (as a proxy for detecting that `receiver` is nothrow constructible.

Similarly for the other overloads.